server/src/lib.rs

```diff
@@ -104,15 +101,15 @@ pub extern "C" fn audioipc_server_new_client(p: *mut c_void) -> PlatformHandleTy
     // We create a pair of connected unix domain sockets. One socket is
     // registered with the reactor core, the other is returned to the
     // caller.
```

Comment:
a pair of connected unix domain sockets or windows named pipe ...

Reply:
Thanks, will fix!